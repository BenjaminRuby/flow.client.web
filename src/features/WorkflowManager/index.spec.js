import React from "react";
import { WorkflowManagerContainer as WorkflowManager } from "./index";

jest.mock("@boomerang/boomerang-components", () => ({
  NoDisplay: "NoDisplay",
  LoadingAnimation: "LoadingAnimation",
  notify: "notify",
  Notification: "Notification"
}));

const mockfn = jest.fn();

const props = {
  actions: {
    fetch: () => new Promise(() => {}),
    reset: mockfn
  },
  history: {},
  match: {},
  tasks: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  tasksActions: {
    fetch: mockfn,
    reset: mockfn
  },
  teams: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  workflow: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  workflowActions: {
    reset: mockfn,
    create: mockfn,
    setHasUnsavedWorkflowUpdates: mockfn
  },
  workflowRevision: {
    isFetching: false,
    status: "success",
    error: "",
    data: []
  },
  workflowRevisionActions: {
    fetch: mockfn,
    create: mockfn,
    reset: mockfn
  }
};

describe("WorkflowManager --- Snapshot", () => {
  it("Capturing Snapshot of WorkflowManager", () => {
    const { baseElement } = rtlRouterRender(<WorkflowManager {...props} />);
    expect(baseElement).toMatchSnapshot();
  });
});
