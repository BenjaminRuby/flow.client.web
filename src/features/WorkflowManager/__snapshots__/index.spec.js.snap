// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkflowManager --- Snapshot Capturing Snapshot of WorkflowManager 1`] = `
<body>
  <div>
    <div
      class="c-workflow-designer"
    />
  </div>
</body>
`;
